idd/Energy+.idd.in

```diff
@@ -58168,10 +57975,16 @@ Coil:Heating:WaterToAirHeatPump:EquationFit,
         \required-field
         \type object-list
         \object-list QuadvariateFunctions
-   A7;  \field Heating Power Consumption Curve Name
+   A7,  \field Heating Power Consumption Curve Name
         \required-field
         \type object-list
         \object-list QuadvariateFunctions
+   A8;  \field Part Load Fraction Correlation Curve Name
+        \note quadratic curve = a + b*PLR + c*PLR**2
+        \note cubic curve = a + b*PLR + c*PLR**2 + d*PLR**3
+        \note PLR = part load ratio (heat load/steady state capacity)
+        \type object-list
+        \object-list UnivariateFunctions
```

Comment:
`\required-field` is missing here.

https://github.com/NREL/EnergyPlus/blob/f1db7f411c004a7e0122a0cf77bc9c03d3a59183/src/EnergyPlus/WaterToAirHeatPumpSimple.cc#L610-L620